### PR TITLE
feat(examples/vmseries_ha): Add vmseries HA Common VPC Peering example

### LINKED ISSUE #202: Example: vmseries-ha
We need to bring [this](https://github.com/PaloAltoNetworks/google-cloud-vmseries-ha-tutorial/tree/main) code into our repo

### PR DESCRIPTION
**Pull request by horiagunica**
## Description

Add a new example within GCP for VM-Series HA within Common VPC Peering reference architecture.
This PR closes #202 .

## Motivation and Context

NGFW Phase 2.

## How Has This Been Tested?

Deployed in test environment and followed steps described in README.md .

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes if appropriate.
- [x] All new and existing tests passed.

### CODE REVIEW

File: examples/vmseries_ha/README.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+Configuration committed successfully
+```
+
+## Check access via web UI
+
+Use a web browser to access `https://<MGMT_PUBLIC_IP>` and login with admin and your previously configured password.
+
+## Change the public Loopback public IP Address
+
+For the VM-Series that are backend instance group members of the public-facing loadbalancer - go to Network -> Interfaces -> Loopback and change the value of `1.1.1.1` with the value from the `EXTERNAL_LB_PUBLIC_IP` from the terraform outputs.
```
Comment (sebastianczech) [inline examples/vmseries_ha/README.md:166]: In TF outputs it's used name `lbs_external_ips` for map, which contains key `external-lb`, so maybe could we rephrase it into: 

```suggestion
For the VM-Series that are backend instance group members of the public-facing loadbalancer - go to Network -> Interfaces -> Loopback and change the value of `1.1.1.1` with the value of object `external-lb` from the `lbs_external_ips` in Terraform outputs.
```

File: examples/vmseries_ha/README.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+
+In order to successfuly access the web server hosted on `spoke-1-vm` - also reconfigure the two NAT policies that contain the destination address of `1.1.1.1` with the IP address from `EXTERNAL_LB_PUBLIC_IP` in Policies -> NAT -> `no-nat-lb-healthchecks` + `inbound-app1`
```
Comment (sebastianczech) [inline examples/vmseries_ha/README.md:168]: Maybe let's use `external-lb`:

```suggestion
In order to successfuly access the web server hosted on `spoke-1-vm` - also reconfigure the two NAT policies that contain the destination address of `1.1.1.1` with the IP address from `lbs_external_ips ` from Terraform outputs in Policies -> NAT -> `no-nat-lb-healthchecks` + `inbound-app1`
```

File: examples/vmseries_ha/README.md [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+
+## Check traffic from spoke VMs
+
+The firewalls are bootstrapped with a generic `allow any` policy just for demo purposes along with an outboud SNAT policy to allow Inernet access from spoke VMs.
+
+SSH to one of the spoke VMs using GCP IAP and gcloud command and test connectivity :
+
+
+```
+gcloud compute ssh spoke1-vm
```
Comment (sebastianczech) [inline examples/vmseries_ha/README.md:178]: Is user is changing region and prefix, then we need to extend a little bit that command:

```suggestion
gcloud compute ssh <NAME-PREFIX-VALUE-FROM-TFVARS>spoke1-vm --zone=<NAME-OF-THE-ZONE-FROM-TFVARS>
```

File: examples/vmseries_ha/README.md [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+## Check traffic towards the test HTTP web server
+
+From any browser access `http://<EXTERNAL_LB_PUBLIC_IP>`
+
+## Test fail-over
+
+Connect to the spoke VM via gcloud CLI and continously ping a destination on the internet :
+
+```
+gcloud compute ssh spoke1-vm
```
Comment (sebastianczech) [inline examples/vmseries_ha/README.md:199]: Similar as in previous comment: 

```suggestion
gcloud compute ssh <NAME-PREFIX-VALUE-FROM-TFVARS>spoke1-vm --zone=<NAME-OF-THE-ZONE-FROM-TFVARS>
```

**Review by sebastianczech — APPROVED** [review]
Great work, I deployed it in the lab, I tested failover and it works without any issues 👍

**Review by migara — APPROVED** [review]
Great work!

**Pull request merged**